src/samples/index.md

```diff
@@ -218,6 +218,64 @@ including initializer lists, optional `new` and `const`, redirecting constructor
 `factory` constructors, getters, setters, and much more.
 
 
+## Enums
+
+Enums are a way of enumerating a predefined set of values or instances.
```

Comment:
Consider adding

 "in a way which ensures that there cannot be any other instances of that type."